gradle.properties

```diff
@@ -3,6 +3,7 @@ version=1.1.0
 gsonVersion=2.8.6
 junitVersion=5.6.2
 mockitoVersion=3.3.3
-newRelicTelemetryVersion=0.9.0
+newRelicTelemetryVersion=0.11.0
```

Comment:
We need to be careful about these version numbers - 0.11.0 has not been released yet, so this will break the build of anyone who tries to build this before we have released the other module.